central/Dockerfile

```diff
@@ -17,7 +17,11 @@ COPY --from=builder --chown=glowroot:root /tmp/glowroot-central /usr/share/glowr
 
 WORKDIR /usr/share/glowroot-central
 
-RUN chmod a+x /usr/local/bin/docker-entrypoint.sh \
+# need to give group write permission because OpenShift will ignore USER directives
+# and run the POD as a numbered userid which cannot possibly exist on the underlying physical host
+# but which is guaranteed to be a member of the "root" group
+RUN chmod g+w /usr/share/glowroot-central \
+    chmod a+x /usr/local/bin/docker-entrypoint.sh \
```

Comment:
isn't there a && missing?

Reply:
@robinroos can you check this? thx

Reply:
These && seem to be missing indeed. I made a PR that squashed all attempts from this PR here: #943